### ISSUE TITLE
rule: multicol-no-column-rule

### ISSUE THREAD

**Issue by purupurupu** [purupurupu/css-noop-checker#126]
## Rule ID
`multicol-no-column-rule`

## What does this rule detect?
`column-rule`, `column-rule-color`, `column-rule-style`, `column-rule-width`, `column-fill` only apply to multicol containers (elements with `column-count` or `column-width` set). On other elements, they have no effect.

## Example: should warn
```html
<div style="column-rule: 1px solid black;">text</div>
```

## Example: should NOT warn
```html
<div style="column-count: 3; column-rule: 1px solid black;">text</div>
```

## Related CSS Properties
`column-count`, `column-width`, `column-rule`, `column-rule-color`, `column-rule-style`, `column-rule-width`, `column-fill`

## References
- [CSS Multi-column Layout Level 1 §5 - Column Gaps and Rules](https://drafts.csswg.org/css-multicol/#column-rule): applies to multicol containers
- [Firefox inactive-property-helper.js](https://searchfox.org/mozilla-central/source/devtools/server/actors/utils/inactive-property-helper.js) — detects this pattern
